### PR TITLE
Add note about FSOA equip to !fsoa and dpm-advice-mage + update rotation

### PR DESCRIPTION
**Pull request by Uta1105**
Follow greater concentrated blast with a dual wield ability to maintain the critical buff but swap to staff as you do it ability to get 2h autos which deal more damage.

### CODE REVIEW

**Comment by Ferroric** [comment]
Might try and trim this down a little, or @Archken might want to.  Seems a bit wordy for what is being said

**Comment by inferiumRS3** [comment]
In my previous attempt at this I inserted a star character beside every abil that uses DW cast with + staff equip same tick, and then inserted a footer that said * = cast with DW + equip same tick

**Comment by inferiumRS3** [comment]
Message link w/ my file: https://discord.com/channels/534508796639182860/689575078698287152/978729068100980776

F784657F-4D6C-4B5E-8D2C-CDB3F0910CF1

**Comment by Archken** [comment]
Yeah i feel like inferiums solution is ideal. I just dont like how it looks but its one of those things where its really the cleanest way to implement it

So should just do it inferiums way for now and if someone can think of something better then we can go with that in future
